src/data/ha-state-history-data.html

```diff
@@ -1,171 +1,339 @@
-<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
 
 <script>
 (function () {
```

Comment:
Since we're doing ES2015 with `let` and `const`, we can replace IIFE with just `{ … }`